chromadb/test/property/test_add.py

```diff
@@ -75,6 +118,8 @@ def test_add(
     )
 
 
+# Hypothesis struggles to generate large record sets so we explicitly create
+# a large record set
 def create_large_recordset(
```

Comment:
Feel like we could add some more randomization in here. For example, all embeddings are the same - this is guaranteed to produce a bad HNSW graph.

Reply:
Agreed, I tried to replace this with hypothesis but still need to do some munging, I cut myself a task